### PR TITLE
Save classifications to Cassandra

### LINKED ISSUE #987: sending transcriptions/classifications/markings directly to Cassandra
@rogerhutchings and @VVH would like to run a beta of annoTate starting July 1st. I'd like to be able to read in the transcriptions directly from Cassandra. @camallen thinks this is doable

### PR DESCRIPTION
**Pull request by edpaget**
Duplicate classification records into Cassandra for aggregation engine

Closes #987

### CODE REVIEW

**Comment by edpaget** [comment]
I put in a PR for my Cequel changes https://github.com/cequel/cequel/pull/249

File: app/models/cassandra/classification.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  column :gold_standard, :boolean
+  column :expert_classifier, :boolean
+  column :metadata, :text
+  column :annotations, :text
+
+  after_create :save_subject
+
+  def self.from_ar_model(classification)
+    attrs = classification.attributes.dup
+    attrs.delete("id")
```
Comment (camallen) [inline app/models/cassandra/classification.rb]: could the panoptes id be used to link a classification with multiple subjects for say a pairwise comparison?

Comment (edpaget) [inline app/models/cassandra/classification.rb]: Greg's annotation code wasn't using it, and I'm not :100: % sure of the affect of including it as part of the cassandra key. We could add it though.

Comment (camallen) [inline app/models/cassandra/classification.rb]: ok, @ggdhines any use case for including the panoptes-id key to group multi subject classifications together?

Comment (ggdhines) [inline app/models/cassandra/classification.rb]: if you mean multi frame classifications like WW, then no (seems to work fine with just multiple frames). if you mean multiple subjects as with Solar Stormwatch - then maybe. Haven't though about that. Maybe include it to be on the safe side

Comment (camallen) [inline app/models/cassandra/classification.rb]: yeah this is for projects like solar stormwatch - will it break your current schema?

Comment (ggdhines) [inline app/models/cassandra/classification.rb]: yeah - stormwatch will probably break a bunch of things. But will have to deal with that sort of project separately

Comment (camallen) [inline app/models/cassandra/classification.rb]: @ggdhines will including the panoptes id key break things or will the aggregation engine just not run for projects like stormwatch? We're only interested in the former, the later can be punted till we run a project like this.

Comment (ggdhines) [inline app/models/cassandra/classification.rb]: @cam - the id key shouldn't break anything

**Comment by edpaget** [comment]
Whoo!

**Pull request merged**